endpoints-support.md

```diff
@@ -33,13 +33,14 @@ Refer SimpliVity REST API doc for the resource endpoints documentation [HPE Simp
 |<sub>/hosts/{hostId}/cancel_virtual_controller_shutdown</sub>     |POST    |
 |<sub>/hosts/{hostId}/remove_from_federation</sub>                 |POST    |
 |<sub>/hosts/{hostId}/hardware</sub>                               |GET     |
-|<sub>/hosts/{hostId}/shutdown_virtual_controller </sub>           |POST    |
+|<sub>/hosts/{hostId}/shutdown_virtual_controller</sub>           |POST    |
```

Comment:
please line up the | with the others

Reply:
Thanks Ken, Corrected the alignment with others